docs/user/user.yaml

```diff
@@ -27,12 +27,13 @@ paths:
                       { _id: 6255bc080a75adf9223df444, name: Languages },
                       { _id: 6255bc080a75adf443d7644, name: Biology }
                     ]
-                  totalReviews: 10
-                  averageRating: 4.5
+                  totalReviews: { student: 10, tutor: 8 }
```

Comment:
2 endpoints are missing

users/:id/reviews & users/my-profile

also you must show that PATCH users/:id & DELETE users/:id are restricted to Admin

basically you must depict every rout that we have in user.js file